test/CMakeLists.txt

```diff
@@ -25,6 +25,9 @@ string(REPLACE "-Wl,--no-undefined" ""
 
 set(UNIT_TEST_TARGETS "")
 
+string(RANDOM ALPHABET 0123456789 RANDOM_SEED)
```

Comment:
Maybe take LENGTH greater than 5 (default)?
I suppose 8-9 is good enough.

Reply:
Also, it would be nice to be able to set this seed externally, I suppose, since it may affect test shuffling.

Reply:
Fixed length:

```diff
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -25,7 +25,7 @@ string(REPLACE "-Wl,--no-undefined" ""
 
 set(UNIT_TEST_TARGETS "")
 
-string(RANDOM ALPHABET 0123456789 RANDOM_SEED)
+string(RANDOM LENGTH 9 ALPHABET 0123456789 RANDOM_SEED)
 message(STATUS "Random seed is ${RANDOM_SEED}")
 
 make_lua_path(LUATEST_LUA_PATH
```